### PR TITLE
Update PF6_alpha_bug_request.md

### PR DESCRIPTION
**Pull request by andrew-ronaldson**
updates bug template

### CODE REVIEW

**Review by srambach — APPROVED** [review]
L🐞TM

**Review by mcoker — APPROVED** [review]
🤘👨‍🎤🎸

File: .github/ISSUE_TEMPLATE/PF6_alpha_bug_request.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-name: PF6 alpha bug
+name: PF6 bug
 about: Report a bug discovered while testing the PatternFly 6 alphas
```
Comment (mcoker) [inline .github/ISSUE_TEMPLATE/PF6_alpha_bug_request.md:3]: Do you want to remove "alpha" from this line, too?

```suggestion
about: Report a bug discovered while testing PatternFly 6
```

**Review by evwilkin — APPROVED** [review]
Bravo 👏

**Pull request closed without merging**